src/NLog.Web/NLogHttpModule.cs

```diff
@@ -41,16 +42,26 @@ private void BeginRequestHandler(object sender, EventArgs args)
         {
             if (BeginRequest != null)
             {
-                BeginRequest(sender, args);
+                OnBeginRequest((sender as HttpApplication)?.Context);
```

Comment:
Please change this into:
```c#
BeginRequest?.Invoke(sender, args);
OnBeginRequest((sender as HttpApplication)?.Context);
```

Reply:
oh of course, silly, forgot nobody is going to subscribe at some point

Reply:
committed

Reply:
Yes one could consider marking the entire `NLogHttpModule` as obsolete at some point, and guide people to use the minimal `NLogBufferingTargetWrapperModule` (without EventHandler-logic)